### ISSUE TITLE
GUI auto-opens at seemingly random intervals on OSX

### ISSUE THREAD

**Issue by cwbuecheler** [keybase/client#3493]
Keybase GUI Version: 1.0.16-20160714110022+dc4f1af
OSX El Capitan

apologies if this is a dupe -- I searched around but couldn't find anything. Periodically my Keybase GUI just springs open for no apparent reason. I'll be in an unrelated program and then all of a sudden, there's the Keybase GUI (which has also resized to an odd window size that is vertically larger than my screen, and sometime has logged me out, but sometimes not).

Is this perhaps happening because the app auto-updated?

**Comment by reedlaw** [comment]
Yes, I believe this is the same behavior I see on Linux and Windows systems as well. Whenever keybase updates the GUI pops up stealing focus from whatever I'm doing.
